test/e2e/storage/volume_metrics.go

```diff
@@ -66,10 +66,6 @@ var _ = utils.SIGDescribe(framework.WithSerial(), "Volume metrics", func() {
 		ns = f.Namespace.Name
 		var err error
 
-		// The tests below make various assumptions about the cluster
-		// and the underlying storage driver and therefore don't pass
-		// with other kinds of clusters and drivers.
-		e2eskipper.SkipUnlessProviderIs("gce", "gke", "aws")
 		e2epv.SkipIfNoDefaultStorageClass(ctx, c)
```

Comment:
the kind cluster doesn't have a default sc, so this e2e will be skipped too.

Reply:
Is the cleanest solution refactoring these tests to always rely on a test driver like csi-hostpath instead of provisioner of default SC? 

What do you think?

Especially since it seems like a couple of these volume metrics tests don't work on any job... [see here](https://github.com/kubernetes/kubernetes/pull/126345#issuecomment-2263593103).

Reply:
> the kind cluster doesn't have a default sc, so this e2e will be skipped too.

That shouldn't be true? https://github.com/kubernetes-sigs/kind/blob/88616663c9364c0b5b867f8c775d9827d64541b1/pkg/build/nodeimage/const_storage.go#L141

Reply:
however we don't have metrics server, there were some issues installing it and it's not part of conformance.